exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/QueryWrapper.java

```diff
@@ -83,12 +84,13 @@ public QueryResult run(final WorkManager workManager, final WebUserConnection we
     float usagePercent = getHeapUsage();
 
     // Wait until the query execution is complete or there is error submitting the query
-    logger.debug("Wait until the query execution is complete or there is error submitting the query");
+    if (logger.isDebugEnabled()) {
```

Comment:
No need do this check, it's done internally.

Reply:
Ok.